src/uniprotkb/components/protein-data-views/FeaturesTableView.tsx

```diff
@@ -1,10 +1,10 @@
 import React, { Fragment, useCallback, useState, FC } from 'react';
 import { TemplateResult } from 'lit-html';
 import ProtvistaDatatable from 'protvista-datatable';
+import { Feature as VariantFeature } from 'protvista-variation-adapter/dist/es/variants';
```

Comment:
Not sure if that should be imported from the dist folder. Couldn't there be a way to expose that at the library level directly?